benchmarks/run.py

```diff
@@ -105,8 +105,11 @@ def null_missing_email():
 
 
 def rand_date():
-    r = random.randrange
-    return f'{r(1900, 2020)}-{r(0, 12)}-{r(0, 32)}T{r(0, 24)}:{r(0, 60)}:{r(0, 60)}'
+    ts = random.randrange(
```

Comment:
I don't agree with this change:
* I've used the same data (good bad or ugly) to do the benchmarks since their inception, I don't want to change it unless I have to
* the point is that some of these should be invalid, os check that the validation library catches them

Reply:
> I've used the same data (good bad or ugly) to do the benchmarks since their inception, I don't want to change it unless I have to

Then it would make sense to seed the rng, so that everyone gets the same cases.json?

> the point is that some of these should be invalid, os check that the validation library catches them

Alright, I thought the invalid dates was an oversight and not intended.